src/Assignments/ProblemIframe.tsx

```diff
@@ -71,6 +79,79 @@ export const ProblemIframe: React.FC<ProblemIframeProps> = ({
         setHeight(`${scrollHeight}px`);
     };
 
+    const formDataToObject = (formData: FormData) => {
+        let object:any = {};
+        // @ts-ignore
```

Comment:
This was a temporary solution, we should see if there is a solution in which we don't have to ts-ignore

Also if we ts-ignore something we should put a comment above as to why we are doing so

Reply:
added down stream iterator comment